files/entrypoint.sh

```diff
@@ -35,6 +35,10 @@ if [ "$PROXY_ALLOW_REMOTE_USER" ]; then
     echo 'underscores_in_headers on;' > /etc/nginx/conf.d/underscores_in_headers.conf
 fi
 
+if [ "$NGINX_HTTP_PORT" ]; then
+    sed -i "s/\<80\>/$NGINX_HTTP_PORT/" /etc/nginx/conf.d/default-http.conf
```

Comment:
Should probably match on more than just "80" so that if someone sets `worker_processes 80;` it doesn't fork bomb their machine when they set the port to 60000.

Reply:
Made it check if the variable is actually a number so it guards against weird input.  Otherwise there is only one place that a number gets used in `/etc/nginx/conf.d/default-http.conf`.